### ISSUE TITLE
version 1.1.15: Errors and warnings output when parsing file

### ISSUE THREAD

**Issue by glodieu** [raml-org/raml-js-parser-2#666]
Parsing different set of raml files, output messages continues after updating parser to version 1.1.15.
```
[xmldom warning]	unclosed xml attribute
[xmldom warning]	attribute space is required"method"!!
[xmldom error]	element parse error: Error: invalid tagName:https:
@#[line:1,col:1]
[xmldom warning]	attribute "”ok”" missed quot(")!!
@#[line:1,col:1]
[xmldom warning]	attribute "”1.2”" missed quot(")!!
@#[line:1,col:1]
[xmldom warning]	attribute "”56”" missed quot(")!!
@#[line:2,col:3]
[xmldom warning]
[xmldom warning]	attribute space is required"previouspageuri"!!
@#[line:2,col:3]
[xmldom warning]	attribute space is required"nextpageuri"!!
@#[line:2,col:5]
[xmldom error]	element parse error: Error: invalid tagName:http:
@#[line:1,col:1]
```

**Comment by ddenisenko** [comment]
Thaks for letting us know. 

Could you please provide RAML sample?

**Comment by glodieu** [comment]
[attribute_missed_quot.txt](https://github.com/raml-org/raml-js-parser-2/files/814538/attribute_missed_quot.txt)
[attribute_space_is_required.txt](https://github.com/raml-org/raml-js-parser-2/files/814537/attribute_space_is_required.txt)
[element_parse_error.txt](https://github.com/raml-org/raml-js-parser-2/files/814539/element_parse_error.txt)
[unclosed_xml_attribute.txt](https://github.com/raml-org/raml-js-parser-2/files/814540/unclosed_xml_attribute.txt)

**Comment by ddenisenko** [comment]
Will be published in 1.1.16
